test/test_managercli.py

```diff
@@ -1171,9 +1171,19 @@ class TestImportCertCommand(TestCliCommand):
     command_class = managercli.ImportCertCommand
 
     def test_certificates(self):
+        self.cc.is_registered = Mock(return_value=False)
         self.cc.main(["--certificate", "one", "--certificate", "two"])
         self.cc._validate_options()
 
+    def test_registered(self):
+        self.cc.is_registered = Mock(return_value=True)
+        self.cc.main(["--certificate", "one", "--certificate", "two"])
+        try:
+            self.cc._validate_options()
+            self.fail("No exception raised")
```

Comment:
you can make this clearer and shorter now with [assertRaises](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises)

```python
with self.assertRaises(SystemExit) as e:
    self.cc._validate_options()
self.assertEqual(os.EX_USAGE, e.exception.code)
```